docs/source/developer_guide/add_estimators.rst

```diff
@@ -133,10 +133,30 @@ or third party modules that aim for interface compliance with ``sktime``,
 the ``sktime`` test suite can be imported and extended in two ways:
 
 *   importing ``check_estimator``, this will carry out the tests defined in ``sktime``
+
+.. code-block:: python
+
+    >>> from sktime.utils.estimator_checks import check_estimator
+    >>> from sktime.forecasting.naive import NaiveForecaster
+    >>> check_estimator(NaiveForecaster)
+
 *   importing test classes, e.g., ``test_all_estimators.TestAllEstimators`` or
     ``test_all_forecasters.TestAllForecasters``. The imports will be discovered directly
     by ``pytest``. The test suite also be extended by inheriting from the test classes.
 
+.. code-block:: python
```

Comment:
this is not how you would do it if you want to run it via pytest.

pytest runs them if the class is *present* already, it does not need to be run.

Reply:
can you give the code snippet for `pytest`?

Reply:
sth along the lines of

```python

from sktime.tests.test_all_estimators import TestAllEstimators

from mypackage.registry import list_of_all_estimators_getter()

class TestEstimatorsInMyPackage(TestAllEstimators):

    # this overrides the retrieval of estimators to the third party package
    def _all_estimators(self):
        return list_of_all_estimators_getter()

    def test_my_new_custom_test(self, estimator_class, scenario):
        # do sth useful here, fixtures available are as in BaseFixtureGenerator
```

to override fixture generation or add more fixtures, one needs to add `_generate_` functions; decorating a test with a `pytest.mark.parametrize` also works.

Reply:
this is the complicate route, for extending the *test framework*

Reply:
the easier route is sth like

```python

from sktime.utils.estimator_checks import check_estimator

@pytest.mark.parametrize("estimator", ESTIMATOR_LIST)
def test_my_estimators(estimator):
    check_estimator(estimator, return_exception=False)
```

Reply:
I just wanted to document somewhere how to run test for a dedicated estimator, so the last one seems good for that

Reply:
but, this is for a test in an external package.

If you are *developing*, you just call `check_estimator`